src/intl/en/page-layer-2.json

```
@@ -67,8 +67,7 @@
   "layer-2-sidechains-2": "Both scale similarly to layer 2s - they offer lower transaction fees and higher transaction throughput - but have different trust assumptions.",
   "layer-2-more-on-sidechains": "More info on sidechains",
   "layer-2-more-on-validiums": "More info on validiums",
-  "layer-2-sidechains-4": "Some <strong>layer 1 blockchains</strong> have higher throughput and lower transaction fees than Ethereum. These alternative layer 1s have had to <b>sacrifice on security or decentralization</b> in order to achieve higher transactions per second and lower transaction fees.",
-  "layer-2-sidechains-5": "The Ethereum ecosystem is firmly aligned that <strong>layer 2 scaling is the only way to solve the scalability trilemma</strong> while remaining decentralized and secure.",
+  "layer-2-sidechains-4": "Some layer 1 blockchains report higher throughput and lower transaction fees than Ethereum, but generally with trade-offs elsewhere, for example greater hardware requirements for running nodes.",
```

```suggestion
  "layer-2-sidechains-4": "Some layer 1 blockchains report higher throughput and lower transaction fees than Ethereum, but generally with trade-offs elsewhere, for example greater hardware requirements for running nodes.",
```
can we add here a comment that that results in significant centralization and control over the network?  or something like that

---

The arguments around centralizing forces are very nuanced. I think it's better to stay very high level here as different alt-L1's have different trade-offs and so does Ethereum's L2-centric model of scaling.